worldengine/draw.py

```diff
@@ -213,46 +203,34 @@ def draw_riversmap(world, target):
     sea_color = (255, 255, 255, 255)
     land_color = (0, 0, 0, 255)
 
-    for y in range(world.height):#TODO: numpy
+    for y in range(world.height):
         for x in range(world.width):
-            if world.ocean[y, x]:
-                target.set_pixel(x, y, sea_color)
-            else:
-                target.set_pixel(x, y, land_color)
+            target.set_pixel(x, y, sea_color if world.ocean[y, x] else land_color)
```

Comment:
could we use the method `is_ocean` here instead of accessing `world.ocean[y, x]`, that is because we could change the way we store ocean data later